modules/rosa-create-objects.adoc

```diff
@@ -114,7 +114,7 @@ $ rosa create cluster --cluster=<cluster_name> | <cluster_id> [arguments]
 |Block of IP addresses (ipNet) from which pod IP addresses are allocated. Example: `10.128.0.0/14`
 
 |--private
-|Restricts master API endpoint and application routes to direct, private connectivity.
+|Restricts primary API endpoint and application routes to direct, private connectivity.
```

Comment:
This is one of the cases where you changed 'master' to something other than 'control plane'. It makes sense, but want to highlight the change just in case :-).